README.md

```diff
@@ -29,6 +29,14 @@ joining our [public Slack channel.](https://adhocteam-public.herokuapp.com)
 We think a sign of a good developer is one who asks questions sooner, rather
 than later. We are happy to help guide you to the right solution.
 
+On Overengineering
```

Comment:
@omgitsbillryan good catch about the quick tip thing... I'll put in another PR to sync those up.

I thought you were going to say that having an entire section dedicated to _not_ overengineering was itself overengineering. ;) The thing I get stuck on here is that, within reason, building a somewhat elaborate solution to the problems to show off how you know stuff is a reasonable way to respond to them. But it's also a thing that we commonly ding people for, so we clearly have a preference on it. With things like that, I'd prefer to be really explicit about the expectation so that I can ding them without feeling bad.